app/controllers/items/detail.js

```diff
@@ -570,9 +578,18 @@ export default GoodcityController.extend(
       },
 
       async updatePackageType() {
-        const pkgType = await this.get(
-          "packageTypeService"
-        ).userPickPackageType();
+        let pkgType;
+
+        if (this.get("model.isPartOfSet")) {
+          pkgType = await this.get("packageTypeService").userPickPackageType({
+            subsetPackageTypes: this.get(
+              "model.packageSet.code"
+            ).defaultChildPackagesList()
```

Comment:
Does the child package list include the type itself ?

Reply:
yup